tempesta_fw/sock.c

```diff
@@ -50,8 +50,15 @@ static const char *ss_statename[] = {
 };
 #endif
 
+#define SS_LISTEN_VAL		0x0000000000000001UL
+#define SS_LISTEN_VAL_MASK	0x000000000000ffffUL
+#define SS_SOFTIRQ_VAL		0x0000000000010000UL
```

Comment:
> is it correct that they way it's used, the maximum number of these is the number of active CPUs in the system?

Basically, yes. I create a comment. Actually `__ss_active` and the constants implement a counter of current SS down- and upcalls on the fly. This is different from connections counting.